cmd/vicadmin/server.go

```diff
@@ -255,7 +255,8 @@ func (s *server) tailFiles(res http.ResponseWriter, req *http.Request, names []s
 func (s *server) index(res http.ResponseWriter, req *http.Request) {
 	defer trace.End(trace.Begin(""))
 	ctx := context.Background()
-	v := vicadmin.NewValidator(ctx, vchConfig)
+	sess, err := client()
```

Comment:
Actually, install/validator.NewValidator also did more than to create a session only. It will figure out the problem if datacenter is not correctly set.
Would you need that function?